### PR TITLE
[ECR-3034] Update doc for build-in services configuration

### PR DESCRIPTION
**Pull request by bullet-tooth**

### CODE REVIEW

File: src/get-started/java-binding.md [hunk 1]
```diff
@@ -501,11 +501,18 @@ Currently Java Binding includes the following built-in services:
   Time oracle allows user services to access the calendar time supplied by
   validator nodes to the blockchain.
 
-To enable a particular service, include its name in `ejb_app_services.toml`
-configuration file in the EJB App's directory with the following content:
+## Services Definition
```
Comment (MakarovS) [inline src/get-started/java-binding.md]: Add a blank line after this one

File: src/get-started/java-binding.md [hunk 2]
```diff
@@ -501,11 +501,19 @@ Currently Java Binding includes the following built-in services:
...
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: Can we clarify the section name? Services Definition sounds too vague. Maybe "Services Activation" or "Node Configuration". The latter is too broad, as there is way more configuration options than this single file. Do we have, however, a section on the node configuration? If not maybe we shall add it with this sub-section and some reference to the usual three-step configuration process (but we also have extensions to that process, which are documented in App_tutorial)?

Seems unclear and easy to get lost. I'd rename this section and submit an issue to develop a better structure/extend this documentation/update App tutorial.

File: src/get-started/java-binding.md [hunk 3]
```diff
@@ -501,11 +501,19 @@ Currently Java Binding includes the following built-in services:
...
+
+Services are disabled by default. To enable a particular service,
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: I'd rewrite and shorten this paragraph somehow. @Maria-Nosyk , could you help with that please?

Maybe, "No services are enabled on the node by default. `ejb_app_services.toml` configuration file defines the services to enable. This configuration file should be located in a **working directory**, where you are running commands. It consists of two sections: `system_services` and `user_services`."

File: src/get-started/java-binding.md [hunk 4]
```diff
@@ -514,8 +522,29 @@ where possible values for `service-name` are:
...
 
-In case there is no `ejb_app_services.toml` file, only Configuration Service will
-be activated.
+!!! note
+    In case there is no such section,
+    only Configuration Service will be activated.
+
+The `user_services` section enumerates services in form of
+`name = artifact`, where `name` is one-word description of the service
+and `artifact` is full path to the service's artifact.
```
[the remark this answers is not in the document]
Comment (dmitry-timofeev) [reply src/get-started/java-binding.md]: It says 'full' ~ 'absolute'. I'd say it must be absolute unless you want to depend on the application working directory.

File: src/get-started/java-binding.md (hunk 2 above)
Comment (bullet-tooth) [reply src/get-started/java-binding.md]: created ECR-3048

File: src/get-started/java-binding.md [hunk 5]
```diff
@@ -514,8 +523,33 @@ where possible values for `service-name` are:
 - `btc-anchoring` for Anchoring Service.
 - `time` for Time Oracle.
 
-In case there is no `ejb_app_services.toml` file, only Configuration Service will
-be activated.
+!!! note
+    In case there is no such section,
+    only Configuration Service will be activated.
+
+The `user_services` section enumerates services in form of
```
Comment (Maria-Nosyk) [inline src/get-started/java-binding.md]: in the form

File: src/get-started/java-binding.md [hunk 6]
```diff
@@ -514,8 +523,33 @@ where possible values for `service-name` are:
...
+`name = artifact`, where `name` is one-word description of the service
```
Comment (Maria-Nosyk) [inline src/get-started/java-binding.md]: is a one-word description

File: src/get-started/java-binding.md [hunk 7]
```diff
@@ -514,8 +523,33 @@ where possible values for `service-name` are:
...
+and `artifact` is full path to the service's artifact. It must be absolute
```
Comment (Maria-Nosyk) [inline src/get-started/java-binding.md]: is a full path to the artifact of the service.

File: src/get-started/java-binding.md [hunk 8]
```diff
@@ -514,8 +523,33 @@ where possible values for `service-name` are:
...
+!!! note
+    At least one service must be defined
+    in the `[user_services]` section.
+
+```toml
+[user_services]
+service_name1 = "/path/to/service1_artifact.jar"
+```
+
+The sample of `ejb_app_services.toml` file that enables
```
Comment (Maria-Nosyk) [inline src/get-started/java-binding.md]: Below is a sample of the `ejb_app_services.toml` file...

File: src/get-started/java-binding.md [hunk 9]
```diff
@@ -514,8 +523,33 @@ where possible values for `service-name` are:
...
+!!! note
+    At least one service must be defined
+    in the `[user_services]` section.
+
+```toml
+[user_services]
+service_name1 = "/path/to/service1_artifact.jar"
+```
+
+The sample of the `ejb_app_services.toml` file that enables
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: "Below is …" as @Maria-Nosyk recommended seems to be more natural

File: src/get-started/java-binding.md [hunk 10]
```diff
@@ -501,11 +501,20 @@ Currently Java Binding includes the following built-in services:
...
+
+No services are enabled on the node by default. To enable services,
+define them in the `ejb_app_services.toml` configuration file.
+This file is required for a running node. `ejb_app_services.toml`
+should be located in the **working directory** of your project,
+where you run commands.  
+It consists of two sections:
+`system_services` and `user_services`.
+
+The optional `system_services` section is used to enable built-in Exonum services.
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: Let's put the required section user_services first, and optional — second.

File: package-lock.json [hunk 11]
```diff
@@ -91,12 +91,12 @@
       "integrity": "sha512-5oNkD/L++l0O6xGXxb1EWS7SivtjfGQlRyxJsYgE0Z495/L81e2h4/d3r969hoPXuFItzNOKMtsXgYG4c7dYvw==",
       "dev": true,
       "requires": {
-        "dot-prop": "4.2.0",
-        "graceful-fs": "4.1.11",
-        "make-dir": "1.1.0",
-        "unique-string": "1.0.0",
-        "write-file-atomic": "2.3.0",
-        "xdg-basedir": "3.0.0"
+        "dot-prop": "^4.1.0",
```
Comment (dmitry-timofeev) [inline package-lock.json]: I am not sure we shall change the dependencies in this branch (unless something isn't working) :-)

**Pull request merged**